scripts/us_epa/ejscreen/ejscreen.py

```diff
@@ -115,25 +132,55 @@ def write_tmcf(outfilename):
 if __name__ == '__main__':
     dfs = {}
     for year in YEARS:
-        print(year)
         columns = CSV_COLUMNS_BY_YEAR[year]
-        # request file
+        #request file
         zip_filename = ZIP_FILENAMES[year]
+
         if zip_filename is not None:
-            response = requests.get(
-                f'https://gaftp.epa.gov/EJSCREEN/{year}/{zip_filename}.zip')
-            with zipfile.ZipFile(io.BytesIO(response.content())) as zfile:
+            if year == '2024':
+                logging.info(f'inside 2024 {zip_filename}')
+                response = requests.get(
+                    f'https://gaftp.epa.gov/EJSCREEN/2024/2.32_August_UseMe/{zip_filename}.zip',
+                    verify=False)
+            elif year == '2023':
+                logging.info(f'inside 2023 {zip_filename}')
+                response = requests.get(
+                    f'https://gaftp.epa.gov/EJSCREEN/2023/2.22_September_UseMe/{zip_filename}.zip',
+                    verify=False)
+            else:
+                response = requests.get(
+                    f'https://gaftp.epa.gov/EJSCREEN/{year}/{zip_filename}.zip',
+                    verify=False)
+            #response = requests.get(f'https://gaftp.epa.gov/EJSCREEN/2023/2.22_September_UseMe/{year}/{zip_filename}.zip',verify=False)
+            with zipfile.ZipFile(io.BytesIO(response.content)) as zfile:
```

Comment:
Before accessing the response content, verify that the response.status_code is 200 to ensure a successful request.